deploy/template.yaml

```diff
@@ -0,0 +1,179 @@
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref DCSResponseTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAccessTokensTable
+      Events:
+        IPVCriUKPassportAPI:
+          Type: Api
+          Properties:
+            RestApiId: !Ref IPVCriUKPassportAPI
+            Path: /credential
+            Method: GET
+
+  IPVDcsAccessTokenFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-dcs-token-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.accesstoken.AccessTokenHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/accesstoken
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Environment:
+        Variables:
+          ENVIRONMENT: !Sub "${Environment}"
+          CRI_PASSPORT_AUTH_CODES_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAuthCodesTable.Arn]]
+          CRI_PASSPORT_ACCESS_TOKENS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAccessTokensTable.Arn]]
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAuthCodesTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAccessTokensTable
+      Events:
+        IPVCriUKPassportAPI:
+          Type: Api
+          Properties:
+            RestApiId: !Ref IPVCriUKPassportAPI
+            Path: /token
+            Method: POST
+
+  IPVPassportFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-passport-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.passport.PassportHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/passport
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Environment:
+        Variables:
+          ENVIRONMENT: !Sub "${Environment}"
+          DCS_RESPONSE_TABLE_NAME: !Select [1, !Split ['/', !GetAtt DCSResponseTable.Arn]]
+          CRI_PASSPORT_ACCESS_TOKENS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAccessTokensTable.Arn]]
+          CRI_PASSPORT_AUTH_CODES_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAuthCodesTable.Arn]]
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref DCSResponseTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAccessTokensTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAuthCodesTable
+      Events:
+        IPVCriUKPassportAPI:
+          Type: Api
+          Properties:
+            RestApiId: !Ref IPVCriUKPassportAPI
+            Path: /passport
+            Method: POST
+
+  IPVDcsStubFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-stub-dcs-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.dcsstub.StubDcsHandler::handleRequest
```

Comment:
Think this one is wrong as well.

uk.gov.di.ipv.cri.passport.stubdcs.StubDcsHandler::handleRequest